src/test/VoteMonitor.Api.IntegrationTests/VoteMonitor.Api.IntegrationTests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyName>VoteMonitor.Api.IntegrationTests</AssemblyName>
+    <PackageId>VoteMonitor.Api.IntegrationTests</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <UserSecretsId>TBR</UserSecretsId>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <ProjectGuid>{0e63cb9a-486a-48f7-91b2-b1e2766088d3}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.6" />
```

Comment:
My only problem with SqlLite is that at the very first point when StoredProcedures will be introduced SqlLite will not be an option anymore.  We could use an in-memory database although I don't think that is supported in EF Core 3.X.
I would be more confident running the tests against SQL server running in a Linux container.

A very nice article for Jimmy Bogard, the father of AutoMapper and MediatR:
https://jimmybogard.com/avoid-in-memory-databases-for-tests/

Reply:
As I mentioned in the opening comment of this PR, I went with SqlLite just because is the middle ground between the memory-based ones and SQL Server. The latter requires more setup and happy to do that in a different PR after some discussions about the CI pipeline too.

Reply:
@czlatea indeed this might be a problem but for now we do not have stored procedures. We could use this tool https://github.com/jbogard/Respawn
For now I think this is good

Reply:
@czalea @idormenco I added Respawn, happy to discuss more on slack